### PR TITLE
i18n(tr): add `sidebar.mdx`

### PR DESCRIPTION
**Pull request by erkanaltinors**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- This pull request adds translated in turkish `sidebar.mdx` file into `tr/guides`

### CODE REVIEW

**Review by HiDeoo — COMMENTED** [review]
Thanks for the translation.

I think this may be missing the changes from #881 which adds details about BCP-47 language tags and uses a BCP-47 language tag with a regional subtag for the "Internationalization" section example.

**Comment by erkanaltinors** [comment]
Hi @HiDeoo thanks for your response. I will add this block into the page.

File: docs/src/content/docs/tr/guides/sidebar.mdx [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+			],
+		},
+	]}
+/>
+
+#### Önbölümde otomatik oluşturulmuş bağlantıları özelleştirmek
+
+Özel sayfalarınızda [`kenar çubuğu` önbölüm alanı](/reference/frontmatter/#sidebar)'nı otomatik oluşturulan bağlantılarınızı özelleştirmek için kullanın.
+
+Kenar çubuğu önbölüm ayarları bir bağlantıya [özel isim](/tr/reference/frontmatter/#label) ya da [etiket](/tr/reference/frontmatter/#badge) eklemenize, kenar çubuğundan bir bağlantı [gizlemenize](/reference/frontmatter/#hidden) ya da [özel sıralama](/reference/frontmatter/#order) tanımlamanıza olanak sağlar.
```
Comment (delucis) [inline docs/src/content/docs/tr/guides/sidebar.mdx:192]: Same here — some of these need updating to include `/tr/`:

```suggestion
Özel sayfalarınızda [`kenar çubuğu` önbölüm alanı](/tr/reference/frontmatter/#sidebar)'nı otomatik oluşturulan bağlantılarınızı özelleştirmek için kullanın.

Kenar çubuğu önbölüm ayarları bir bağlantıya [özel isim](/tr/reference/frontmatter/#label) ya da [etiket](/tr/reference/frontmatter/#badge) eklemenize, kenar çubuğundan bir bağlantı [gizlemenize](/tr/reference/frontmatter/#hidden) ya da [özel sıralama](/tr/reference/frontmatter/#order) tanımlamanıza olanak sağlar.
```

File: docs/src/content/docs/tr/guides/sidebar.mdx [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+			collapsed: true,
+			items: [
+				{ label: 'Bileşenler', link: '/tr/guides/components/' },
+				{ label: 'Uluslararasılaştırma (i18n)', link: '/tr/guides/i18n/' },
+			],
+		},
+	]}
+/>
+
+[Otomatik oluşturulmuş gruplar](#autogenerated-groups) `collapsed` değerinin üst grubuna dikkat eder:
```
Comment (delucis) [inline docs/src/content/docs/tr/guides/sidebar.mdx:437]: This link needs to be to the Turkish slug:
```suggestion
[Otomatik oluşturulmuş gruplar](#otomatik-oluşturulmuş-gruplar) `collapsed` değerinin üst grubuna dikkat eder:
```

**Review by delucis — CHANGES_REQUESTED** [review]
Thanks for the translation @erkanaltinors! Just noticed a few links that need updating, but otherwise looks good to me 👍

**Comment by erkanaltinors** [comment]
Hi @delucis , I will fix them in this week. Thank you for your feedback 🤠

**Comment by delucis** [comment]
Thanks @erkanaltinors! You should be able to click the <kbd>Commit suggestion</kbd> buttons in this PR to accept the suggested fixes.

**Comment by erkanaltinors** [comment]
I've accepted all suggested fixes @delucis 🚀

**Review by delucis — APPROVED** [review]
Awesome! Thanks @erkanaltinors 💫

**Pull request merged**